packages/components/src/molecules/SkuSelector/SkuSelector.tsx

```diff
@@ -0,0 +1,135 @@
+import type { ChangeEventHandler, HTMLAttributes } from 'react'
+import React, { forwardRef, ReactNode } from 'react'
+import { Label } from '../..'
+
+interface SkuProps {
```

Comment:
Kind of nit, but how about renaming this to something like `Option`?
I think the `Props` in the name might be misleading.